code/game/objects/items/devices/scanners.dm

```diff
@@ -124,6 +124,7 @@ REAGENT SCANNER
 	origin_tech = "magnets=1;biotech=1"
 	var/upgraded = 0
 	var/mode = 1;
+	var/timeofdeath = null

```

Comment:
I can't even begin to articulate what is wrong about this.

This isn't showing their time of fucking death, this is asking for the worldtime2text of a null value.